crates/aptos-drop-helper/Cargo.toml

```
@@ -20,3 +20,6 @@ threadpool = { workspace = true }
 
 [dev-dependencies]
 rayon = { workspace = true }
+
+[features]
+no-metrics = []
```

Since Cargo features are additive, it's bad style to have negative features.

```suggestion
metrics = []
```